### PR TITLE
[INLONG-2097][Agent] change wget command

### LINKED ISSUE #2097: [Bug][Docker] error while building tubemq image
### What happened

An error occurred when build tubemq image.

The command is: 

```shell
mvn --batch-mode --update-snapshots -e -V clean package -DskipTests -Pdocker
```

The working directory is the root directory of inlong.

See https://github.com/shink/incubator-inlong/runs/4671539072?check_suite_focus=true

```java
Caused by: com.spotify.docker.client.exceptions.DockerException: The command '/bin/sh -c wget -O "/tmp/protobuf-${PROTOBUF_VERSION}.zip" "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip"     && unzip "/tmp/protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/"     && cd /opt/protobuf-${PROTOBUF_VERSION}     && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install     && rm -f "/tmp/protobuf-${PROTOBUF_VERSION}.zip"' returned a non-zero code: 8
    at com.spotify.plugin.dockerfile.LoggingProgressHandler.handleError (LoggingProgressHandler.java:105)
    at com.spotify.plugin.dockerfile.LoggingProgressHandler.progress (LoggingProgressHandler.java:63)
    at com.spotify.docker.client.DefaultDockerClient$BuildProgressHandler.progress (DefaultDockerClient.java:312)
    at com.spotify.docker.client.ProgressStream.tail (ProgressStream.java:77)
    at com.spotify.docker.client.DefaultDockerClient$ResponseTailReader.call (DefaultDockerClient.java:2754)
    at com.spotify.docker.client.DefaultDockerClient$ResponseTailReader.call (DefaultDockerClient.java:2738)
    at java.util.concurrent.FutureTask.run (FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:624)
    at java.lang.Thread.run (Thread.java:748)
```

### What you expected to happen

All images can be built correctly.

### How to reproduce

Enter the root directory of inlong, execute the following command.

```shell
mvn --batch-mode --update-snapshots -e -V clean package -DskipTests -Pdocker
```

### Environment

- ubuntu: 20.04
- docker: 20.10.12
- java: 1.8.0_261
- maven: 3.8.4

### InLong version

master

### InLong Component

InLong TubeMQ

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by dockerzhang**
### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes #2097

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile [hunk 1]
```diff
@@ -21,8 +21,8 @@ FROM gcc:5.4.0 as protobuf
 RUN apt-get update && apt-get install -y unzip \
     && rm -rf /var/lib/apt/lists/
 ARG PROTOBUF_VERSION=2.5.0
-RUN wget -O "/tmp/protobuf-${PROTOBUF_VERSION}.zip" "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
-    && unzip "/tmp/protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
+RUN curl -LOk "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
+    && unzip "protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
     && cd /opt/protobuf-${PROTOBUF_VERSION} \
     && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install \
     && rm -f "/tmp/protobuf-${PROTOBUF_VERSION}.zip"
```
Comment (shink) [inline inlong-tubemq/tubemq-docker/tubemq-build/Dockerfile:28]: It seems that the zip file is download, but not deleted after unzipping.

```suggestion
RUN curl -Lk "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-${PROTOBUF_VERSION}.zip" \
    -o "/tmp/protobuf-${PROTOBUF_VERSION}.zip"
    && unzip "/tmp/protobuf-${PROTOBUF_VERSION}.zip" -d "/opt/" \
    && cd /opt/protobuf-${PROTOBUF_VERSION} \
    && ./configure --prefix=/usr && make && make DESTDIR=/protobuf/ install \
    && rm -rf "/tmp/*"
```

File: inlong-tubemq/tubemq-docker/tubemq-cpp/Dockerfile [hunk 2]
```diff
@@ -22,11 +22,11 @@ RUN wget https://www.openssl.org/source/openssl-1.1.0f.tar.gz --no-check-certifi
     && tar -xzvf openssl-1.1.0f.tar.gz \
     && rm openssl-1.1.0f.tar.gz && cd openssl-1.1.0f \
     && ./config && make && make install
```
Comment (shink) [inline inlong-tubemq/tubemq-docker/tubemq-cpp/Dockerfile:24]: This place also should be replaced with `curl`.

Comment (shink) [reply inlong-tubemq/tubemq-docker/tubemq-cpp/Dockerfile:24]:
```suggestion
RUN curl -LOk https://www.openssl.org/source/openssl-1.1.0f.tar.gz --no-check-certificate \
    && tar -xzvf openssl-1.1.0f.tar.gz \
    && rm openssl-1.1.0f.tar.gz && cd openssl-1.1.0f \
    && ./config && make && make install
```

**Review by healchow — APPROVED** [review]
+1

**Review by charlely — APPROVED** [review]
LGTM

**Review by gosonzhang — APPROVED** [review]
+1

**Comment by shink** [comment]
+1

**Review by shink — APPROVED** [review]
+1

**Pull request merged**